gen/build_deploy/bash.py

```diff
@@ -701,24 +707,26 @@ def do_create(tag, build_name, reproducible_artifact_path, commit, variant_argum
     """
     # TODO(cmaloney): Build installers in parallel.
     # Variants are sorted for stable ordering.
-    for variant, bootstrap_info in sorted(variant_arguments.items(),
-                                          key=lambda kv: pkgpanda.util.variant_str(kv[0])):
-        with logger.scope("Building installer for variant: ".format(pkgpanda.util.variant_name(variant))):
-            bootstrap_installer_name = '{}installer'.format(pkgpanda.util.variant_prefix(variant))
-            installer_filename = make_installer_docker(variant, all_completes[variant],
-                                                       all_completes[bootstrap_installer_name])
-
-            yield {
-                'channel_path': 'dcos_generate_config.{}sh'.format(pkgpanda.util.variant_prefix(variant)),
-                'local_path': installer_filename
-            }
+    for variant in sorted(variant_arguments.keys(), key=lambda k: pkgpanda.util.variant_str(k)):
+        variant_name = pkgpanda.util.variant_name(variant)
+        bootstrap_installer_name = '{}installer'.format(pkgpanda.util.variant_prefix(variant))
+        bootstrap_util_name = '{}util'.format(pkgpanda.util.variant_prefix(variant))
+        if bootstrap_installer_name not in all_completes:
+            print('WARNING: No installer tree for variant: {}'.format(variant_name))
```

Comment:
The code is currently written so that if you do not include an installer tree with your variant, then it will fail hard:
https://github.com/dcos/dcos/blob/master/gen/build_deploy/bash.py#L707